src/executors/windows.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+# How to author Executors: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors
+description: >
+  An executor preloaded with Visual Studio 2019 plus a number of other
+  development tools.
+parameters:
+  version:
+    type: string
+    description: The image version to use when executing. Defaults to "stable".
+    default: stable
+  shell:
+    type: string
+    description: |
+      The shell to use. Defaults to `bash.exe`
```

Comment:
Is there a reason we don't default to cmd or powershell instead of bash? I don't think we technically support bash on windows and most of our users use cmd. It might be nice to use that by default.